### PR TITLE
Crew monitoring server board

### PR DESCRIPTION
**Pull request by Nimfar11**
## About the PR

Now if the crew monitoring server is destroyed or unavailable, the RND will be able to build it.

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

:cl: Nimfar11

- add: Adds the ability to research and print the crew monitoring server board.

### CODE REVIEW

**Comment by keronshb** [comment]
This needs discussion. There's already powergaming issues with handheld crew monitors, and freely being able to print the computer boards adds to that.

**Comment by Nimfar11** [comment]
It is already possible to print consoles and portable crew monitoring consoles. This board, gives the ability to restore the monitoring work if the server was destroyed or it is not on the map at all. It would be strange not to use the basic mechanics.

**Comment by metalgearsloth** [comment]
> This needs discussion. There's already powergaming issues with handheld crew monitors, and freely being able to print the computer boards adds to that.

I think we should just move the handheld crew monitor later and slot this in instead potentially?

**Comment by keronshb** [comment]
> > This needs discussion. There's already powergaming issues with handheld crew monitors, and freely being able to print the computer boards adds to that.
> 
> I think we should just move the handheld crew monitor later and slot this in instead potentially?

I was thinking instead of allowing it to be printed, a spare could be in tech storage and/or more spares could be ordered from cargo as needed.

**Comment by lzk228** [comment]
you should add gold and maybe plasma to recipe

**Comment by Nimfar11** [comment]
> Wait portable crew monitors can be printed and they're not just limited by vendor??

No, they can print by learning the technology you need for the console and as I suggested for the server. Only there it seems to need gold. So if anything to limit, then the cost of the most handheld crew monitor.

> you should add gold and maybe plasma to recipe

Definitely not for the server, I think it is worth making more expensive for rare resources itself handheld monitor.

File: Resources/Prototypes/Entities/Structures/Machines/lathe.yml [hunk 1]
```diff
@@ -341,6 +341,7 @@
       - BoozeDispenserMachineCircuitboard
       - SodaDispenserMachineCircuitboard
       - TelecomServerCircuitboard
+      - CrewMonitoringServerMachineCircuitboard
```
Comment (keronshb) [inline Resources/Prototypes/Entities/Structures/Machines/lathe.yml:344]: Please add this to cargo as an option to buy instead.

**Review by keronshb — CHANGES_REQUESTED** [review]
I talked it over, we feel that the board should be put into cargo instead. It helps control it more over printing. The crew monitoring console will also be moved in a separate PR.

There should also be a spare mapped onto each map in tech storage as well, if it isn't there already. No need to do it for this PR though.

**Pull request closed without merging**